prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -1107,5 +1107,15 @@ public async Task UpdateDateOfBirthAsync(int enrolleeId, DateTime dateOfBirth)
             enrollee.DateOfBirth = dateOfBirth;
             await _context.SaveChangesAsync();
         }
+
+        public async Task<bool> IsEnrolleeAbsentAsync(int enrolleeId)
+        {
+            var now = DateTime.Now;
```

Comment:
should be `DateTime.UtcNow` if comparing with the `StartTimestamp` and `EndTimestamp` which are stored as UTC time without timezone in database.